doc/build-unix.md

```diff
@@ -61,34 +61,40 @@ Dependency Build Instructions: Ubuntu & Debian
 ----------------------------------------------
 Build requirements:
 
-	sudo apt-get install build-essential libtool autotools-dev autoconf pkg-config libssl-dev libevent-dev
+        sudo apt-get install build-essential libtool autotools-dev autoconf pkg-config libssl-dev libevent-dev
```

Comment:
Nit: these whitespaces are not consistent throughout the document.

Some code parts are wrapped with \```, while others are not, and the number of spaces seems to be different (e.g. "To build": 0, Build requirements" has more than "To build with Qt 4...").